src/utils/spo.ts

```diff
@@ -1453,5 +1456,213 @@ export const spo = {
     const webProperties: WebProperties = await request.get<WebProperties>(requestOptions);
 
     return webProperties;
+  },
+
+  /**
+  * Gets file properties by server relative file url
+  * Returns a ListItemAllFields object
+  * @param webUrl Web url
+  * @param url The url of the list
+  * @param logger The logger object
+  * @param verbose If in verbose mode
+  */
+  async getFileAsListItemByUrl(webUrl: string, url: string, logger?: Logger, verbose?: boolean): Promise<any> {
```

Comment:
let's now recheck if we may reuse this method in `spo file get` command

Reply:
Technically we could do this, but it doesn't cover it all.
The operation of `spo file get` consists of building the request incrementally, based on the parameters that are provided. if we wanted to implement `getFileAsListItemByUrl` here we would have to undo some of it and make sure it ends up with the same part again.
I leave it up to you whether we should do this or not